### DISCUSSION TITLE
m300

### DISCUSSION THREAD

**Discussion by 3dfixer** [bigtreetech/BIGTREETECH-TouchScreenFirmware#1510] [Q&A]
when in marlin mode M300 makes beep. when in tft mode, nothing. is this a bug/issue?
how do i get tft to play tones to alert me of events in gcode, like pause to drop nut into printed object at right layer height

**Comment by 3dfixer** [comment]
and where the hell is the search button??

**Reply by kisslorand** [reply]
Upper left corner
